src/to_chars.cpp

```diff
@@ -913,7 +913,8 @@ format. Returns an iterator pointing past-the-end of the decimal representation.
 */
 char *to_chars(char *first, const char *last, double value) {
   static_cast<void>(last); // maybe unused - fix warning
-  if (value <= -0) {
+  bool negative = value <= -0;
```

Comment:
I assume it's `-0` instead of `0` for a reason? I am curious what's going on there (and it might be worth commenting, though I know you didn't change this :)).

Reply:
BTW it is almost surely wrong. You are correct that it is not my code. Let me patch it up.